scripts/stats/cam/exevs_cam_nam_firewxnest_grid2obs_stats.sh

```diff
@@ -116,7 +116,7 @@ echo $obfound
 if [ $obfound = 1 -a $fcstnum -gt 0 ]
 then
 
-  run_metplus.py $PARMevs/metplus_config/${COMPONENT}/${VERIF_CASE}/stats/PointStat_fcstNAM_FIREWXNEST_obsPREPBUFR.conf $PARMevs/metplus_config/machine.conf
+  run_metplus.py $PARMevs/metplus_config/stats/${COMPONENT}/${VERIF_CASE}/PointStat_fcstNAM_FIREWXNEST_obsPREPBUFR.conf $PARMevs/metplus_config/machine.conf
```

Comment:
Nitpicking, but for lines 119, 131, and 134, lets use `${STEP}` in splace of `stats`.